forest/cli/src/cli/snapshot_fetch.rs

```diff
@@ -407,4 +487,28 @@ mod test {
             assert!(replace_extension_url(case.0.try_into().unwrap(), case.1).is_err())
         });
     }
+
+    #[test]
+    fn download_with_aria2_test_wrong_checksum() -> Result<()> {
+        let url = "https://github.com/ChainSafe/forest/raw/07f9b397324f63711fd587fa2cff52a20fcb9c51/.github/forest_logo.png";
```

Comment:
From my experience in my former project, Github infrastructure is not that reliable to use in tests. How about using a self-spawned server to make our tests independent (and able to be run offline)?